.gitignore

```diff
@@ -34,3 +34,6 @@ ex_typesense-*.tar
 # Misc.
 .elixir_ls
 **.DS_Store
+
+# openapi generator files
+/openapi/generator/
```

Comment:
The generated API code should be in `/lib/` folder and we shouldn't ignore them.